fractions/swagger/src/main/resources/modules/com/fasterxml/jackson/swagger/module.xml

```diff
@@ -1,18 +1,17 @@
 <module xmlns="urn:jboss:module:1.3" name="com.fasterxml.jackson" slot="swagger">
   <resources>
-    <artifact name="com.fasterxml.jackson.core:jackson-core:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.core:jackson-annotations:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.core:jackson-databind:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-joda:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-guava:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-xml:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:${version.swagger.jackson}"/>
+    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-joda:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-guava:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-xml:${version.com.fasterxml.jackson}"/>
   </resources>
 
   <dependencies>
     <module name="javax.ws.rs.api"/>
     <module name="javax.api"/>
+    <module name="com.fasterxml.jackson.core.jackson-core" export="true"/>
```

Comment:
Based on the limited reading, I noticed that I will need to export the APIs for the Swagger module to consume. Let me know if this is supposed to be done differently.